### PR TITLE
added UserMapper, UserController, UserResponceDto and config classes

### PR DESCRIPTION
**Pull request by less08**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -42,11 +43,38 @@
...
+            <groupId>javax.servlet.jsp</groupId>
+            <artifactId>javax.servlet.jsp-api</artifactId>
+            <version>2.3.3</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>jstl</groupId>
+            <artifactId>jstl</artifactId>
+            <version>1.2</version>
+        </dependency>
```
Comment (vicire) [inline pom.xml]: Do we need these two dependencies?

File: src/main/java/spring/intro/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    public String inject() {
+        User bob = new User();
+        bob.setEmail("bob@gmail.com");
+        bob.setPassword("bobpass");
+        User alice = new User();
+        alice.setEmail("alice@gmail.com");
+        alice.setPassword("alicepass");
+        userService.add(bob);
+        userService.add(alice);
+        return "All users were injected";
```
Comment (vicire) [inline src/main/java/spring/intro/controller/UserController.java:43]: Optional. I don`t know was it a strong requirement to create exactly 4 users, so maybe 2 will be enough.

**Review by vicire — CHANGES_REQUESTED** [review]

**Review by vicire — APPROVED** [review]

File: src/main/java/dto/UserResponseDto.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package dto;
+
+public class UserResponseDto {
+    private Long id;
+    private String email;
+    private String password;
```
Comment (dmytro-hryhoruk) [inline src/main/java/dto/UserResponseDto.java]: Let's remove password from response Dto

File: src/main/java/spring/intro/controller/UserController.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    @GetMapping("/{userId}")
+    public UserResponseDto get(@PathVariable("userId") Long userId) {
+        User user = userService.getById(userId);
+        return userMapper.createDtoFromUser(user);
+    }
+
+    @GetMapping("/")
```
Comment (dmytro-hryhoruk) [inline src/main/java/spring/intro/controller/UserController.java]:
```suggestion
    @GetMapping
```

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**